<!doctype html><html lang=en-us><head><script type=application/javascript>var doNotTrack=!1;doNotTrack||(window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-162151853-1","auto"),ga("send","pageview"))</script><script async src=https://www.google-analytics.com/analytics.js></script><meta name=description content="Why OpenMS? Powerful n-dimensional arrays. Numerical computing tools. Interoperable. Performant. Open source."><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta http-equiv=x-ua-compatible content="ie=edge"><title>openms</title><link rel=icon href=/images/favicon.png><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Lato:400,900"><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css><link rel=stylesheet type=text/css href=/theme-css/fresh/core.min.3997f86943b5337a6aab5eac5003a41b9a03e1159a7e779d8e99f5c1353ad503.css integrity="sha256-OZf4aUO1M3pqq16sUAOkG5oD4RWafnedjpn1wTU61QM="><link rel=stylesheet href=/css/theme-css/code-highlight.css.min.60358f6aca833402d029f1625f9d78ac9331cb988429c4541f68207cfe3894ed.css integrity="sha256-YDWPasqDNALQKfFiX514rJMxy5iEKcRUH2ggfP44lO0="><link rel=stylesheet href=/css/theme-css/content.css.min.9cd6eb8cf77772925445b567376a88573a3e3760d10b6548bbb0d9d8fadee9f2.css integrity="sha256-nNbrjPd3cpJURbVnN2qIVzo+N2DRC2VIu7DZ2Pre6fI="><link rel=stylesheet href=/css/theme-css/keyfeatures.css.min.f1cec535025edc9c051477db79f3425c8f1a819ca95ddfe7c1aa7d00e67657bb.css integrity="sha256-8c7FNQJe3JwFFHfbefNCXI8agZypXd/nwap9AOZ2V7s="><link rel=stylesheet href=/css/theme-css/news.css.min.4bffc1f3e8e1dfedfa26d21423b0f206b10be408002e80d7628649afd6bb6a9d.css integrity="sha256-S//B8+jh3+36JtIUI7DyBrEL5AgALoDXYoZJr9a7ap0="><link rel=stylesheet href=/css/theme-css/notices.css.min.a1ba05c782ca3a59361d6e284f0c2e1ff14e5a17a87057974212c60aa30fed2c.css integrity="sha256-oboFx4LKOlk2HW4oTwwuH/FOWheocFeXQhLGCqMP7Sw="><link rel=stylesheet href=/css/theme-css/panel.css.min.d42639c23c159bedb13ee97ac0c12439a6ff5bf2e557fc976b54856abe4a616b.css integrity="sha256-1CY5wjwVm+2xPul6wMEkOab/W/LlV/yXa1SFar5KYWs="><link rel=stylesheet href=/css/theme-css/posts.css.min.6d27826ef3e7467b2bc97db6a0068283b47dddb32f6bf29504911aea6c237dd7.css integrity="sha256-bSeCbvPnRnsryX22oAaCg7R93bMva/KVBJEa6mwjfdc="><link rel=stylesheet href=/css/theme-css/shortcuts.css.min.be3212556604caeb9849fba8c3971af183b6cf8dbaa121509d756f51049b5b0c.css integrity="sha256-vjISVWYEyuuYSfuow5ca8YO2z426oSFQnXVvUQSbWww="><link rel=stylesheet href=/css/theme-css/styles.css.min.f2547d534742323363c1716409b361bc024aa7fb1d6c1887d3e08dcdcf65c08b.css integrity="sha256-8lR9U0dCMjNjwXFkCbNhvAJKp/sdbBiH0+CNzc9lwIs="><link rel=stylesheet href=/css/theme-css/tables.css.min.fb97ebea26ca35733dddd7051622133c047b532bf1aa066155c69a118d445803.css integrity="sha256-+5fr6ibKNXM93dcFFiITPAR7UyvxqgZhVcaaEY1EWAM="><link rel=stylesheet href=/css/theme-css/teams.css.min.fcfb8db7476eab78560aefc0efcec4366740523e08279ec26c568590c69002fc.css integrity="sha256-/PuNt0duq3hWCu/A787ENmdAUj4IJ57CbFaFkMaQAvw="><link rel=stylesheet href=/css/theme-css/vars.css.min.ce4f6cd2245b2484e4446657011e6acbecb59c4a2db6c9c8c55326783f504611.css integrity="sha256-zk9s0iRbJITkRGZXAR5qy+y1nEottsnIxVMmeD9QRhE="><link rel=stylesheet href=/css/theme-css/videos.css.min.a4b57fbdc20f9eef545905202e188c480604a56cb2e60febea38b297bf680c9c.css integrity="sha256-pLV/vcIPnu9UWQUgLhiMSAYEpWyy5g/r6jiyl79oDJw="><link rel=stylesheet href=/css/css/casestudies.css.min.62be4a9853ed7cc0e0aef0523c1c6f6b0e8c21b24459cf425310d03779f7c4bf.css integrity="sha256-Yr5KmFPtfMDgrvBSPBxvaw6MIbJEWc9CUxDQN3n3xL8="><link rel=stylesheet href=/css/css/keyfeatures.css.min.d1a2192a588e3582e843c571601797f903043d70e07f2a18cb80c120fba8c21c.css integrity="sha256-0aIZKliONYLoQ8VxYBeX+QMEPXDgfyoYy4DBIPuowhw="><link rel=stylesheet href=/css/css/mailchimp.css.min.9898339e77f305768f551fb53b214e818e03828d124c95b1ad8ef34554c314f1.css integrity="sha256-mJgznnfzBXaPVR+1OyFOgY4Dgo0STJWxrY7zRVTDFPE="><link rel=stylesheet href=/css/css/shell.css.min.dc27513b7cb36f4334b06b9266a297ddd8079a7e100cdc6cbf97e37aa1a0d7ec.css integrity="sha256-3CdRO3yzb0M0sGuSZqKX3dgHmn4QDNxsv5fjeqGg1+w="><link rel=stylesheet href=/css/css/tabs.css.min.96924996dbf5cd31d28ae036f0ba9ede7c429c46c71b331c68856fbfd928bb80.css integrity="sha256-lpJJltv1zTHSiuA28Lqe3nxCnEbHGzMcaIVvv9kou4A="><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://openms.org/images/logo.svg"><meta name=twitter:title content="Applications"><meta name=twitter:description content="Why OpenMS? Powerful n-dimensional arrays. Numerical computing tools. Interoperable. Performant. Open source."></head><body><nav id=nav class="navbar is-fresh is-transparent no-shadow" role=navigation aria-label="main navigation"><div class="container is-max-widescreen"><div class=navbar-brand><a class=navbar-item href=/><img class=navbar-logo src=/images/openms-logo.png alt="openms logo"><div class=navbar-logo-text></div></a><a role=button class=navbar-burger aria-label=menu aria-expanded=false data-target=navbar-menu><span aria-hidden=true></span>
<span aria-hidden=true></span>
<span aria-hidden=true></span></a></div><div id=navbar-menu class="navbar-menu is-static"><div class=navbar-end><a href=/install class="navbar-item is-secondary">Install</a>
<a href=https://OpenMS.org/doc/stable class="navbar-item is-secondary">Documentation</a>
<a href=/community class="navbar-item is-secondary">Community</a>
<a href=/about class="navbar-item is-secondary">About Us</a>
<a href=/contribute class="navbar-item is-secondary">Contribute</a>
<a href=/publications class="navbar-item is-secondary">Publications</a>
<a href=/help-request class=navbar-item><span class="button rounded secondary-btn raised" style=background-color:#0c2a60;color:azure>Help Request</span></a></div></div></div></nav><nav id=navbar-clone class="navbar is-fresh is-transparent" role=navigation aria-label="main navigation"><div class="container is-max-widescreen"><div class=navbar-brand><a class=navbar-item href=/><img class=navbar-logo src=/images/openms-logo.png alt="openms logo"><div class=navbar-logo-text></div></a><a role=button class=navbar-burger aria-label=menu aria-expanded=false data-target=cloned-navbar-menu><span aria-hidden=true></span>
<span aria-hidden=true></span>
<span aria-hidden=true></span></a></div><div id=cloned-navbar-menu class="navbar-menu is-fixed"><div class=navbar-end><a href=/install class="navbar-item is-secondary">Install</a>
<a href=https://OpenMS.org/doc/stable class="navbar-item is-secondary">Documentation</a>
<a href=/community class="navbar-item is-secondary">Community</a>
<a href=/about class="navbar-item is-secondary">About Us</a>
<a href=/contribute class="navbar-item is-secondary">Contribute</a>
<a href=/publications class="navbar-item is-secondary">Publications</a>
<a href=/help-request class=navbar-item><span class="button signup-button rounded secondary-btn raised">Help Request</span></a></div></div></div></nav><section class="content-padding flex-row"><div class=shortcuts-container><div class=shortcuts-title><img src=/images/icons/list-bulleted.svg>On this page</div><div id=shortcuts></div></div><div class=content-container><div class=columns><div class="column is-centered-tablet-portrait"><h1 class="title section-title">Applications</h1><h5 class="subtitle is-5 is-muted"></h5><div class=divider></div></div></div><div class=post-list><article><div class=post-title><a href=/application/diametalyzer/>DATA-INDEPENDENT ACQUISITION METABOLOMICS ANALYZER (DIAMETALYZER)</a></div><div class=post-authors></div><div class=post-summary>DIAMetAlyzer is a KNIME workflow which integrates DDA and targeted DIA analysis for metabolomics, which allows for false-discovery rate estimation based a target-decoy approach (see figure 1). It performs DDA based candidate identification and constructs a target/decoy assay library, which is used for DIA target extraction and statistical validation (FDR estimation)
Fig. 1: DIAMetAlyzer â€“ pipeline for assay library generation and targeted analysis with statistical validation
Candidate identification includes feature detection, adduct grouping and accurate mass search.</div></article></div><div class=post-list><article><div class=post-title><a href=/application/openpepxl/>PROTEIN-PROTEIN CROSS-LINKING (OPENPEPXL)</a></div><div class=post-authors></div><div class=post-summary>OpenPepXL: an open source peptide cross-link identification tool OpenPepXL is a protein-protein cross-link identification tool implemented in C++ as part of OpenMS. It works with all uncleavable labeled and label-free cross-linkers but not (yet) with cleavable ones.
Requirements The current version of OpenPepXL, version 1.1 is available as part of OpenMS 2.5. Installers for Windows, MacOS and Linux can be found here. OpenPepXL can be used effectively on a desktop computer with 16BG of memory.</div></article></div><div class=post-list><article><div class=post-title><a href=/application/mhcquant/>QUANTIFICATION AND IDENTIFICATION OF MHC PEPTIDES (MHCQUANT)</a></div><div class=post-authors></div><div class=post-summary>MHCquant: Identify and quantify peptides from mass spectrometry raw data
MHCquant is an analysis pipeline used for quantitative processing of data dependent (DDA) peptidomics data.
It was specifically designed to analyze immunopeptidomics data, which deals with the analysis of affinity-purified, unspecifically cleaved peptides that have recently been discussed intensively in the context of cancer vaccines. (https://www.nature.com/articles/ncomms13404)
The workflow is based on the OpenMS C++ framework for computational mass spectrometry.</div></article></div><div class=post-list><article><div class=post-title><a href=/application/diaproteomics/>QUANTITATIVE DATA INDEPENDENT PROTEOMICS (DIAPROTEOMICS)</a></div><div class=post-authors></div><div class=post-summary>DIAproteomics: Automated quantitative analysis of DIA proteomics mass spectrometry measurements DIAproteomics is a bioinformatics analysis pipeline used for quantitative processing of data independent (DIA) proteomics data.
The workflow is based on the OpenSwathWorkflow for SWATH-MS proteomic data. DIA RAW files (mzML) serve as inputs and library search is performed based on a given input spectral library. If specified internal retention time standards (irts) will be used to align library and DIA measurements into the same retention time space.</div></article></div><div class=post-list><article><div class=post-title><a href=/application/rnpxl/>RNA-PROTEIN CROSS-LINKING (RNPXL)</a></div><div class=post-authors></div><div class=post-summary>RNPxl has been integrated into OpenMS and also the Proteome Discoverer Community Nodes. No custom installer is required anymore. RNPxl has been integrated into OpenMS and also the Proteome Discoverer Community Nodes. No custom installer is required anymore.
Requirements: HCD data acquired on a high-resolution MS Developed and tested on orbitrap instruments MS and MS/MS need to be centroided (either on acquisition, conversion or in a workflow using the TOPP tool HiResPeakPicker) For additional information on sample handling and acquisition, as well as a detailed step-by-step tutorial, please refer to the supplementary material of the publication.</div></article></div><div class=post-list><article><div class=post-title><a href=/application/metaprosip/>STABLE ISOTOPE PROBING (METAPROSIP)</a></div><div class=post-authors></div><div class=post-summary>MetaProSIP: automated inference of elemental fluxes in microbial communities ðŸ›ˆ &nbsp;Info MetaProSIP has been fully integrated into OpenMS. No additional installer is required anymore. Update: MetaProSIP now supports labeling experiments with heavy water (D, 18O)
Download an updated KNIME workflow: here
Advanced workflow including mass recalibration, alignment, and ID pooling: here
Requirements High-Resolution MS, CID, or HCD MS2 Developed and tested on orbitrap instruments MS1 and MS2 need to be centroided (either on acquisition, conversion or in a workflow using the TOPP tool HiResPeakPicker) For additional information on sample handling and acquisition please refer to the original MetaProSIP publication.</div></article></div><div class=post-list><article><div class=post-title><a href=/application/flashdeconv/>ULTRA-FAST MS1/MS2 DECONVOLUTION FOR TOP-DOWN PROTEOMICS</a></div><div class=post-authors></div><div class=post-summary>FLASHDECONV 2.0 BETA+, FINALLY WITH A GUI! Finally a GUI is here. You can find the GUI command in [OpenMS path]/bin folder. Go to [OpenMS path]/bin and run FLASHDeconvWizard! FLASHDeconv 2.0 beta+ works for MS1 and MS2 spectral deconvolution and feature deconvolution. It supports various output formats (e.g., *.tsv, *.mzML, *.msalign, and *.feature). FLASHDeconv 2.0 stable version will be officially integrated in OpenMS 2.7.0 released in near future. FLASHDeconv 2.0 beta+ also supports TopPIC identification better than the previous version, by generating all msalign and feature files for TopPIC inputs.</div></article></div></div></section><div id=backtotop><a href=# id=backtotop-color></a></div><footer id=footer class=footer><div class="container is-max-widescreen"><div id=footer-columns class=columns><div class=footer-logo-column><p style=font-weight:bolder;font-size:xx-large;color:#f0f8ff>OpenMS</p></div><div class=link-column><div class=footer-column><div class=footer-header></div><ul class=link-list><li class=link-list><a class=footer-link href=/install>Install</a></li><li class=link-list><a class=footer-link href=https://OpenMS.org/doc/stable>Documentation</a></li><li class=link-list><a class=footer-link href=/citing-OpenMS>Citing OpenMS</a></li><li class=link-list><a class=footer-link href=https://OpenMS.org/neps/roadmap.html>Roadmap</a></li></ul></div></div><div class=link-column><div class=footer-column><div class=footer-header></div><ul class=link-list><li class=link-list><a class=footer-link href=/about>About us</a></li><li class=link-list><a class=footer-link href=/community>Community</a></li><li class=link-list><a class=footer-link href=/contribute>Contribute</a></li><li class=link-list><a class=footer-link href=/code-of-conduct>Code of conduct</a></li></ul></div></div><div class=link-column><div class=footer-column><div class=footer-header></div><ul class=link-list><li class=link-list><a class=footer-link href=/gethelp>Get help</a></li><li class=link-list><a class=footer-link href=/terms>Terms of use</a></li><li class=link-list><a class=footer-link href=/privacy>Privacy</a></li><li class=link-list><a class=footer-link href=/press-kit>Press kit</a></li></ul></div></div><div class=footer-actions><p>Sign up for the latest openms news, resources,<br>and more</p><form action="https://openms.us4.list-manage.com/subscribe/post?u=5ddd0d1d6e807900a8212481a&id=287fa4253c" method=post id=mc-embedded-subscribe-form name=mc-embedded-subscribe-form class="validate sign-up-container" target=_blank novalidate><div class=sign-up-image><svg class="icon mail-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M22 6c0-1.1-.9-2-2-2H4c-1.1.0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1.0 2-.9 2-2V6zm-2 0-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z"/></svg></div><input type=email name=EMAIL class="required email sign-up-input" id=mce-EMAIL aria-label="Input for email, press enter to submit" onkeypress='(event.which===13||event.keyCode===13||event.key==="Enter")&&sendThankYou()'><div class=submission-instructions>Press Enter</div><button class=signup-button onclick=sendThankYou() aria-label=Submit><svg class="icon sent-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M2.01 21 23 12 2.01 3 2 10l15 2-15 2z"/></svg></button><div id=mce-responses class=clear><div class=response id=mce-error-response style=display:none></div><div class=response id=mce-success-response style=display:none></div></div><div style=position:absolute;left:-5000px aria-hidden=true><input type=text name=b_5ddd0d1d6e807900a8212481a_287fa4253c tabindex=-1></div><div class=clear><input type=submit value=Subscribe name=subscribe id=mc-embedded-subscribe class=button style=display:none></div></form><div class=thank-you>Thank you! &#127881;</div><nav class="level is-mobile"><div class=social-media-icons><a class=level-item href=https://github.com/OpenMS/OpenMS aria-label=https://github.com/OpenMS/OpenMS><svg class="icon github-icon" viewBox="0 0 24 24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 .297c-6.63.0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577.0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93.0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176.0.0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22.0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22.0 1.606-.015 2.896-.015 3.286.0.315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/></svg></a><a class=level-item href=https://www.youtube.com/channel/UCguIL9NZ7ybWK5WQ53qbHng aria-label=https://www.youtube.com/channel/UCguIL9NZ7ybWK5WQ53qbHng><svg class="icon youtube-icon" viewBox="0 0 24 24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M23.498 6.186A3.016 3.016.0 0021.376 4.05C19.505 3.545 12 3.545 12 3.545s-7.505.0-9.377.505A3.017 3.017.0 00.502 6.186C0 8.07.0 12 0 12s0 3.93.502 5.814a3.016 3.016.0 002.122 2.136c1.871.505 9.376.505 9.376.505s7.505.0 9.377-.505a3.015 3.015.0 002.122-2.136C24 15.93 24 12 24 12s0-3.93-.502-5.814zM9.545 15.568V8.432L15.818 12l-6.273 3.568z"/></svg></a><a class=level-item href=https://twitter.com/OpenMS_team aria-label=https://twitter.com/OpenMS_team><svg class="icon twitter-icon" viewBox="0 0 24 24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M23.953 4.57a10 10 0 01-2.825.775 4.958 4.958.0 002.163-2.723c-.951.555-2.005.959-3.127 1.184A4.92 4.92.0 0011.78 8.288C7.69 8.095 4.067 6.13 1.64 3.162A4.822 4.822.0 00.974 5.637c0 1.71.87 3.213 2.188 4.096A4.904 4.904.0 01.934 9.117v.06a4.923 4.923.0 003.946 4.827 4.996 4.996.0 01-2.212.085 4.936 4.936.0 004.604 3.417A9.867 9.867.0 011.17 19.611c-.39.0-.779-.023-1.17-.067a13.995 13.995.0 007.557 2.209c9.053.0 13.998-7.496 13.998-13.985.0-.21.0-.42-.015-.63A9.935 9.935.0 0024 4.59z"/></svg></a></div></nav><div class=copyright>&copy; 2022 OpenMS. All rights reserved.</div></div></div></div></footer><script src=https://code.jquery.com/jquery-3.6.0.min.js></script>
<script type=text/javascript src=/js/bundle.min.js></script>
<script src=https://cdn.jsdelivr.net/npm/uikit@3.7.6/dist/js/uikit.min.js></script>
<link rel=stylesheet href=https://cdn.jsdelivr.net/npm/uikit@3.7.6/dist/css/uikit.min.css><script type=text/javascript>setupShortcuts(maxLevel=2)</script><script type=application/javascript>var doNotTrack=!1;doNotTrack||(window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-162151853-1","auto"),ga("send","pageview"))</script><script async src=https://www.google-analytics.com/analytics.js></script>
<script defer data-domain=openms.org src=https://views.scientific-python.org/js/script.js></script></body></html>