<!doctype html><html lang=en-us><head><script type=application/javascript>var doNotTrack=!1;doNotTrack||(window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-162151853-1","auto"),ga("send","pageview"))</script><script async src=https://www.google-analytics.com/analytics.js></script><meta name=description content="Why OpenMS? Mass Spectrometry. Visualizations. Open source."><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><meta http-equiv=x-ua-compatible content="ie=edge"><title>openms</title><link rel=icon href=/images/favicon.png><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Lato:400,900"><link rel=stylesheet href=https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css><link rel=stylesheet type=text/css href=/theme-css/fresh/core.min.19a8edfc02a0e3ee9689bf2258b4b67599b4f114853619a0666f482ff7447d08.css integrity="sha256-Gajt/AKg4+6Wib8iWLS2dZm08RSFNhmgZm9IL/dEfQg="><link rel=stylesheet href=/css/theme-css/buttons.css.min.ea06123b74a7c5b381789a4cdcd12f4664d9800594f39e5f815bce90a842cfbf.css integrity="sha256-6gYSO3SnxbOBeJpM3NEvRmTZgAWU855fgVvOkKhCz78="><link rel=stylesheet href=/css/theme-css/code-highlight.css.min.60358f6aca833402d029f1625f9d78ac9331cb988429c4541f68207cfe3894ed.css integrity="sha256-YDWPasqDNALQKfFiX514rJMxy5iEKcRUH2ggfP44lO0="><link rel=stylesheet href=/css/theme-css/content.css.min.9cd6eb8cf77772925445b567376a88573a3e3760d10b6548bbb0d9d8fadee9f2.css integrity="sha256-nNbrjPd3cpJURbVnN2qIVzo+N2DRC2VIu7DZ2Pre6fI="><link rel=stylesheet href=/css/theme-css/keyfeatures.css.min.f1cec535025edc9c051477db79f3425c8f1a819ca95ddfe7c1aa7d00e67657bb.css integrity="sha256-8c7FNQJe3JwFFHfbefNCXI8agZypXd/nwap9AOZ2V7s="><link rel=stylesheet href=/css/theme-css/news.css.min.4bffc1f3e8e1dfedfa26d21423b0f206b10be408002e80d7628649afd6bb6a9d.css integrity="sha256-S//B8+jh3+36JtIUI7DyBrEL5AgALoDXYoZJr9a7ap0="><link rel=stylesheet href=/css/theme-css/nfcore.css.min.f5c6dbcdfd95db2bd76af6d717e60fb37ae8f1c5a8fba49034ddb05c96c08081.css integrity="sha256-9cbbzf2V2yvXavbXF+YPs3ro8cWo+6SQNN2wXJbAgIE="><link rel=stylesheet href=/css/theme-css/notices.css.min.a1ba05c782ca3a59361d6e284f0c2e1ff14e5a17a87057974212c60aa30fed2c.css integrity="sha256-oboFx4LKOlk2HW4oTwwuH/FOWheocFeXQhLGCqMP7Sw="><link rel=stylesheet href=/css/theme-css/panel.css.min.d42639c23c159bedb13ee97ac0c12439a6ff5bf2e557fc976b54856abe4a616b.css integrity="sha256-1CY5wjwVm+2xPul6wMEkOab/W/LlV/yXa1SFar5KYWs="><link rel=stylesheet href=/css/theme-css/posts.css.min.e7dff313685b42f324f4175b7cf821d6466d893b69dd0c57b3bb45a5b4603333.css integrity="sha256-59/zE2hbQvMk9BdbfPgh1kZtiTtp3QxXs7tFpbRgMzM="><link rel=stylesheet href=/css/theme-css/pyopenms.css.min.1022f579da3a3da86e90eeb8190dbcd6f4187d0d0a7b99de240c7ae79c8eba67.css integrity="sha256-ECL1edo6PahukO64GQ281vQYfQ0Ke5neJAx655yOumc="><link rel=stylesheet href=/css/theme-css/quick-facts.css.min.970a61079cf92eca5ef12d11c88f08d5bba2e2b8ee95684319a6014fca0a411b.css integrity="sha256-lwphB5z5Lspe8S0RyI8I1bui4rjulWhDGaYBT8oKQRs="><link rel=stylesheet href=/css/theme-css/shortcuts.css.min.4c5f83f3f141bd50906185dae0ac3f3f91b3c85006673e949e7b98eea2104a9a.css integrity="sha256-TF+D8/FBvVCQYYXa4Kw/P5GzyFAGZz6UnnuY7qIQSpo="><link rel=stylesheet href=/css/theme-css/styles.css.min.bedbb499785163c67c55bdec2c1277770764cac19d9bb3076d2ce5daae78b529.css integrity="sha256-vtu0mXhRY8Z8Vb3sLBJ3dwdkysGdm7MHbSzl2q54tSk="><link rel=stylesheet href=/css/theme-css/tables.css.min.fb97ebea26ca35733dddd7051622133c047b532bf1aa066155c69a118d445803.css integrity="sha256-+5fr6ibKNXM93dcFFiITPAR7UyvxqgZhVcaaEY1EWAM="><link rel=stylesheet href=/css/theme-css/teams.css.min.fcfb8db7476eab78560aefc0efcec4366740523e08279ec26c568590c69002fc.css integrity="sha256-/PuNt0duq3hWCu/A787ENmdAUj4IJ57CbFaFkMaQAvw="><link rel=stylesheet href=/css/theme-css/vars.css.min.7d30e28fb6091030dccc39f6fe80ce0070e11f902decc10681f0f85970cdca23.css integrity="sha256-fTDij7YJEDDczDn2/oDOAHDhH5At7MEGgfD4WXDNyiM="><link rel=stylesheet href=/css/theme-css/videos.css.min.a4b57fbdc20f9eef545905202e188c480604a56cb2e60febea38b297bf680c9c.css integrity="sha256-pLV/vcIPnu9UWQUgLhiMSAYEpWyy5g/r6jiyl79oDJw="><link rel=stylesheet href=/css/css/casestudies.css.min.62be4a9853ed7cc0e0aef0523c1c6f6b0e8c21b24459cf425310d03779f7c4bf.css integrity="sha256-Yr5KmFPtfMDgrvBSPBxvaw6MIbJEWc9CUxDQN3n3xL8="><link rel=stylesheet href=/css/css/heroitems.css.min.64921902533d0d1519d4452490715d192e8b1aabbfb8a5beb093aeb4d4e8e7e9.css integrity="sha256-ZJIZAlM9DRUZ1EUkkHFdGS6LGqu/uKW+sJOutNTo5+k="><link rel=stylesheet href=/css/css/keyfeatures.css.min.6659fa5dcf4e2d537227c01facddb5de0a1e39cd00b0392c17477b0546954f90.css integrity="sha256-Zln6Xc9OLVNyJ8AfrN213goeOc0AsDksF0d7BUaVT5A="><link rel=stylesheet href=/css/css/mailchimp.css.min.9898339e77f305768f551fb53b214e818e03828d124c95b1ad8ef34554c314f1.css integrity="sha256-mJgznnfzBXaPVR+1OyFOgY4Dgo0STJWxrY7zRVTDFPE="><link rel=stylesheet href=/css/css/shell.css.min.dc27513b7cb36f4334b06b9266a297ddd8079a7e100cdc6cbf97e37aa1a0d7ec.css integrity="sha256-3CdRO3yzb0M0sGuSZqKX3dgHmn4QDNxsv5fjeqGg1+w="><link rel=stylesheet href=/css/css/tabs.css.min.96924996dbf5cd31d28ae036f0ba9ede7c429c46c71b331c68856fbfd928bb80.css integrity="sha256-lpJJltv1zTHSiuA28Lqe3nxCnEbHGzMcaIVvv9kou4A="><link rel=stylesheet href=/css/css/terminal.css.min.213c720a99726175cce0b18ad0e1c44859bf842612fbab8daae9d3d1bd029ff7.css integrity="sha256-ITxyCplyYXXM4LGK0OHESFm/hCYS+6uNqunT0b0Cn/c="><meta name=twitter:card content="summary_large_image"><meta name=twitter:image content="https://openms.org/images/logo.svg"><meta name=twitter:title content="Publications"><meta name=twitter:description content="2021  A. Marcu, L. Bichmann, L. Kuchenbecker, D. J. Kowalewski, L. K. Freudenmann, L. Backert, L. Mühlenbruch, A. Szolek, M. Lübke, P. Wagner, and others, “HLA Ligand Atlas: a benign reference of HLA-presented peptides to improve T-cell-based cancer immunotherapy,” Journal for immunotherapy of cancer, vol. 9, iss. 4, 2021.   [Bibtex] @article{marcu2021hla, title={HLA Ligand Atlas: a benign reference of HLA-presented peptides to improve T-cell-based cancer immunotherapy}, author={Marcu, Ana and Bichmann, Leon and Kuchenbecker, Leon and Kowalewski, Daniel Johannes and Freudenmann, Lena Katharina and Backert, Linus and M{\&#34;"></head><body><nav id=nav class="navbar is-fresh is-transparent no-shadow" role=navigation aria-label="main navigation"><div class="container is-max-widescreen"><div class=navbar-brand><a class=navbar-item href=/><img class=navbar-logo src=/images/openms-logo.png alt="openms logo"><div class=navbar-logo-text></div></a><a role=button class=navbar-burger aria-label=menu aria-expanded=false data-target=navbar-menu><span aria-hidden=true></span>
<span aria-hidden=true></span>
<span aria-hidden=true></span></a></div><div id=navbar-menu class="navbar-menu is-static"><div class=navbar-end><a href=/install class="navbar-item is-secondary">Install</a>
<a href=https://openms.readthedocs.io/en/latest/ class="navbar-item is-secondary">Documentation</a>
<a href=/training class="navbar-item is-secondary">Training</a>
<a href=/community class="navbar-item is-secondary">Community</a>
<a href=/about class="navbar-item is-secondary">About Us</a>
<a href=/contribute class="navbar-item is-secondary">Contribute</a><div class="navbar-item has-dropdown is-hoverable"><a class=navbar-link>Use Cases</a><div class=navbar-dropdown><a href=/applications class=navbar-item>Applications</a>
<a href=/publications class=navbar-item>Publications</a></div></div><a href=/help-request class=navbar-item><span class="button rounded secondary-btn raised" style=background-color:var(--colorPrimaryDarkBtn);color:azure>Help Request</span></a></div></div></div></nav><nav id=navbar-clone class="navbar is-fresh is-transparent" role=navigation aria-label="main navigation"><div class="container is-max-widescreen"><div class=navbar-brand><a class=navbar-item href=/><img class=navbar-logo src=/images/openms-logo.png alt="openms logo"><div class=navbar-logo-text></div></a><a role=button class=navbar-burger aria-label=menu aria-expanded=false data-target=cloned-navbar-menu><span aria-hidden=true></span>
<span aria-hidden=true></span>
<span aria-hidden=true></span></a></div><div id=cloned-navbar-menu class="navbar-menu is-fixed"><div class=navbar-end><a href=/install class="navbar-item is-secondary">Install</a>
<a href=https://openms.readthedocs.io/en/latest/ class="navbar-item is-secondary">Documentation</a>
<a href=/training class="navbar-item is-secondary">Training</a>
<a href=/community class="navbar-item is-secondary">Community</a>
<a href=/about class="navbar-item is-secondary">About Us</a>
<a href=/contribute class="navbar-item is-secondary">Contribute</a><div class="navbar-item has-dropdown is-hoverable"><a class=navbar-link>Use Cases</a><div class=navbar-dropdown><a href=/applications class=navbar-item>Applications</a>
<a href=/publications class=navbar-item>Publications</a></div></div><a href=/help-request class=navbar-item><span class="button signup-button rounded secondary-btn raised">Help Request</span></a></div></div></div></nav><section class="content-padding flex-row"><div class=shortcuts-container><div class=shortcuts-title><img src=/images/icons/list-bulleted.svg>On this page</div><div id=shortcuts></div></div><div class=content-container><div class=columns><div class="column is-centered-tablet-portrait"><h1 class="title section-title">Publications</h1><h5 class="subtitle is-5 is-muted"></h5><div class=divider></div></div></div><div class=post-meta></div><div class=post-content><h3 id=2021>2021</h3><ul><li>A. Marcu, L. Bichmann, L. Kuchenbecker, D. J. Kowalewski, L. K. Freudenmann, L. Backert, L. Mühlenbruch, A. Szolek, M. Lübke, P. Wagner, and others, “HLA Ligand Atlas: a benign reference of HLA-presented peptides to improve T-cell-based cancer immunotherapy,” <em>Journal for immunotherapy of cancer</em>, vol. 9, iss. 4, 2021.</li></ul><details><summary style="padding:0 0 0 25px" style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{marcu2021hla,
title={HLA Ligand Atlas: a benign reference of HLA-presented peptides to improve T-cell-based cancer immunotherapy},
author={Marcu, Ana and Bichmann, Leon and Kuchenbecker, Leon and Kowalewski, Daniel Johannes and Freudenmann, Lena Katharina and Backert, Linus and M{\"u}hlenbruch, Lena and Szolek, Andr{\'a}s and L{\"u}bke, Maren and Wagner, Philipp and others},
journal={Journal for immunotherapy of cancer},
volume={9},
number={4},
year={2021},
publisher={BMJ Publishing Group}
}
</code></pre></details><hr><ul><li>C. Dai, A. Füllgrabe, J. Pfeuffer, E. Solovyeva, J. Deng, P. Moreno, S. Kamatchinathan, D. J. Kundu, N. George, S. Fexova, and others, “A proteomics sample metadata representation for multiomics integration, and big data analysis,” <em>bioRxiv</em>, 2021.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{dai2021proteomics,
title={A proteomics sample metadata representation for multiomics integration, and big data analysis},
author={Dai, Chengxin and F{\"u}llgrabe, Anja and Pfeuffer, Julianus and Solovyeva, Elizaveta and Deng, Jingwen and Moreno, Pablo and Kamatchinathan, Selvakumar and Kundu, Deepti Jaiswal and George, Nancy and Fexova, Silvie and others},
journal={bioRxiv},
year={2021},
publisher={Cold Spring Harbor Laboratory}
}
</code></pre></details><hr><h3 id=2020>2020</h3><ul><li>R. Starke, K. Oliphant, N. Jehmlich, S. S. Schäpe, T. Sachsenberg, O. Kohlbacher, E. Allen-Vercoe, and M. von Bergen, “Tracing incorporation of heavy water into proteins for species-specific metabolic activity in complex communities,” <em>Journal of Proteomics</em>, p. 103791, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{starke2020tracing,
title={Tracing incorporation of heavy water into proteins for species-specific metabolic activity in complex communities},
author={Starke, Robert and Oliphant, Kaitlyn and Jehmlich, Nico and Sch{\"a}pe, Stephanie Serena and Sachsenberg, Timo and Kohlbacher, Oliver and Allen-Vercoe, Emma and von Bergen, Martin},
journal={Journal of Proteomics},
pages={103791},
year={2020},
publisher={Elsevier}
}
</code></pre></details><hr><ul><li>O. Alka, T. Sachsenberg, L. Bichmann, J. Pfeuffer, H. Weisser, S. Wein, E. Netz, M. Rurik, O. Kohlbacher, and H. Röst, “OpenMS and KNIME for Mass Spectrometry Data Processing,” in <em>Processing Metabolomics and Proteomics Data with Open Software</em>, , 2020, p. 201–231.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@incollection{alka2020openms,
title={OpenMS and KNIME for Mass Spectrometry Data Processing},
author={Alka, Oliver and Sachsenberg, Timo and Bichmann, Leon and Pfeuffer, Julianus and Weisser, Hendrik and Wein, Samuel and Netz, Eugen and Rurik, Marc and Kohlbacher, Oliver and R{\"o}st, Hannes},
booktitle={Processing Metabolomics and Proteomics Data with Open Software},
pages={201--231},
year={2020}
}
</code></pre></details><hr><ul><li>S. Wein, B. Andrews, T. Sachsenberg, H. Santos-Rosa, O. Kohlbacher, T. Kouzarides, B. A. Garcia, and H. Weisser, “A computational platform for high-throughput analysis of RNA sequences and modifications by mass spectrometry,” Nature communications, vol. 11, iss. 1, p. 1–12, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{wein2020computational,
title={A computational platform for high-throughput analysis of RNA sequences and modifications by mass spectrometry},
author={Wein, Samuel and Andrews, Byron and Sachsenberg, Timo and Santos-Rosa, Helena and Kohlbacher, Oliver and Kouzarides, Tony and Garcia, Benjamin A and Weisser, Hendrik},
journal={Nature communications},
volume={11},
number={1},
pages={1--12},
year={2020},
publisher={Nature Publishing Group}
}
</code></pre></details><hr><ul><li>M. Rurik, O. Alka, F. Aicheler, and O. Kohlbacher, “Metabolomics Data Processing Using OpenMS,” in Computational Methods and Data Analysis for Metabolomics, Springer, 2020, p. 49–60.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@incollection{rurik2020metabolomics,
title={Metabolomics Data Processing Using OpenMS},
author={Rurik, Marc and Alka, Oliver and Aicheler, Fabian and Kohlbacher, Oliver},
booktitle={Computational Methods and Data Analysis for Metabolomics},
pages={49--60},
year={2020},
publisher={Springer}
}
</code></pre></details><hr><ul><li>K. Jeong, J. Kim, M. Gaikwad, S. N. Hidayah, L. Heikaus, H. Schlüter, and O. Kohlbacher, “FLASHDeconv: Ultrafast, high-quality feature deconvolution for top-down proteomics,” Cell Systems, vol. 10, iss. 2, p. 213–218, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{jeong2020flashdeconv,
title={FLASHDeconv: Ultrafast, high-quality feature deconvolution for top-down proteomics},
author={Jeong, Kyowon and Kim, Jihyung and Gaikwad, Manasi and Hidayah, Siti Nurul and Heikaus, Laura and Schl{\"u}ter, Hartmut and Kohlbacher, Oliver},
journal={Cell Systems},
volume={10},
number={2},
pages={213--218},
year={2020},
publisher={Elsevier}
}
</code></pre></details><hr><ul><li>L. Bichmann, S. Gupta, G. Rosenberger, L. Kuchenbecker, T. Sachsenberg, P. Ewels, O. Alka, J. Pfeuffer, O. Kohlbacher, and H. Röst, “DIAproteomics: A Multifunctional Data Analysis Pipeline for Data-Independent Acquisition Proteomics and Peptidomics,” Journal of Proteome Research, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bichmann2020diaproteomics,
title={DIAproteomics: A Multifunctional Data Analysis Pipeline for Data-Independent Acquisition Proteomics and Peptidomics},
author={Bichmann, Leon and Gupta, Shubham and Rosenberger, George and Kuchenbecker, Leon and Sachsenberg, Timo and Ewels, Phil and Alka, Oliver and Pfeuffer, Julianus and Kohlbacher, Oliver and Röst, Hannes},
journal={Journal of Proteome Research},
year={2020},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>S. Kutuzova, P. Colaianni, H. Rost, T. Sachsenberg, O. Alka, O. Kohlbacher, B. Burla, F. Torta, L. Schrubbers, M. Kristensen, and others, “SmartPeak automates targeted and quantitative metabolomics data processing,” Analytical chemistry, vol. 92, iss. 24, p. 15968–15974, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{kutuzova2020smartpeak,
title={SmartPeak automates targeted and quantitative metabolomics data processing},
author={Kutuzova, Svetlana and Colaianni, Pasquale and Rost, Hannes and Sachsenberg, Timo and Alka, Oliver and Kohlbacher, Oliver and Burla, Bo and Torta, Federico and Schrubbers, Lars and Kristensen, Mette and others},
journal={Analytical chemistry},
volume={92},
number={24},
pages={15968--15974},
year={2020},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>A. Stützer, L. M. Welp, M. Raabe, T. Sachsenberg, C. Kappert, A. Wulf, A. M. Lau, S. David, A. Chernev, K. Kramer, and others, “Analysis of protein-DNA interactions in chromatin by UV induced cross-linking and mass spectrometry,” Nature communications, vol. 11, iss. 1, p. 1–12, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{stutzer2020analysis,
title={Analysis of protein-DNA interactions in chromatin by UV induced cross-linking and mass spectrometry},
author={St{\"u}tzer, Alexandra and Welp, Luisa M and Raabe, Monika and Sachsenberg, Timo and Kappert, Christin and Wulf, Alexander and Lau, Andy M and David, Stefan-Sebastian and Chernev, Aleksandar and Kramer, Katharina and others},
journal={Nature communications},
volume={11},
number={1},
pages={1--12},
year={2020},
publisher={Nature Publishing Group}
}
</code></pre></details><hr><ul><li>E. Netz, T. M. Dijkstra, T. Sachsenberg, L. Zimmermann, M. Walzer, T. Monecke, R. Ficner, O. Dybkov, H. Urlaub, and O. Kohlbacher, “OpenPepXL: An open-source tool for sensitive identification of cross-linked peptides in XL-MS,” Molecular & Cellular Proteomics, vol. 19, iss. 12, p. 2157–2168, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{netz2020openpepxl,
title={OpenPepXL: An open-source tool for sensitive identification of cross-linked peptides in XL-MS},
author={Netz, Eugen and Dijkstra, Tjeerd MH and Sachsenberg, Timo and Zimmermann, Lukas and Walzer, Mathias and Monecke, Thomas and Ficner, Ralf and Dybkov, Olexandr and Urlaub, Henning and Kohlbacher, Oliver},
journal={Molecular \& Cellular Proteomics},
volume={19},
number={12},
pages={2157--2168},
year={2020},
publisher={ASBMB}
}
</code></pre></details><hr><ul><li>L. Nothias, D. Petras, R. Schmid, K. Dührkop, J. Rainer, A. Sarvepalli, I. Protsyuk, M. Ernst, H. Tsugawa, M. Fleischauer, and others, “Feature-based molecular networking in the GNPS analysis environment,” Nature methods, vol. 17, iss. 9, p. 905–908, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{nothias2020feature,
title={Feature-based molecular networking in the GNPS analysis environment},
author={Nothias, Louis-F{\'e}lix and Petras, Daniel and Schmid, Robin and D{\"u}hrkop, Kai and Rainer, Johannes and Sarvepalli, Abinesh and Protsyuk, Ivan and Ernst, Madeleine and Tsugawa, Hiroshi and Fleischauer, Markus and others},
journal={Nature methods},
volume={17},
number={9},
pages={905--908},
year={2020},
publisher={Nature Publishing Group}
}
</code></pre></details><hr><ul><li>T. Scheidt, O. Alka, H. Gonczarowska-Jorge, W. Gruber, F. Rathje, M. Dell’Aica, M. Rurik, O. Kohlbacher, R. P. Zahedi, F. Aberger, and others, “Phosphoproteomics of short-term hedgehog signaling in human medulloblastoma cells,” Cell Communication and Signaling, vol. 18, iss. 1, p. 1–18, 2020.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{scheidt2020phosphoproteomics,
title={Phosphoproteomics of short-term hedgehog signaling in human medulloblastoma cells},
author={Scheidt, Tamara and Alka, Oliver and Gonczarowska-Jorge, Humberto and Gruber, Wolfgang and Rathje, Florian and Dell’Aica, Margherita and Rurik, Marc and Kohlbacher, Oliver and Zahedi, Ren{\'e} P and Aberger, Fritz and others},
journal={Cell Communication and Signaling},
volume={18},
number={1},
pages={1--18},
year={2020},
publisher={BioMed Central}
}
</code></pre></details><hr><h3 id=2019>2019</h3><ul><li>J. Pfeuffer, T. Sachsenberg, T. M. Dijkstra, O. Serang, K. Reinert, and O. Kohlbacher, “EPIFANY-A method for efficient high-confidence protein inference,” Journal of proteome research, p. 734327, 2019.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{pfeuffer2019epifany,
title={EPIFANY-A method for efficient high-confidence protein inference},
author={Pfeuffer, Julianus and Sachsenberg, Timo and Dijkstra, Tjeerd MH and Serang, Oliver and Reinert, Knut and Kohlbacher, Oliver},
journal={Journal of proteome research},
pages={734327},
year={2019},
publisher={Cold Spring Harbor Laboratory}
}
</code></pre></details><hr><ul><li>N. Hulstaert, J. Shofstahl, T. Sachsenberg, M. Walzer, H. Barsnes, L. Martens, and Y. Perez-Riverol, “ThermoRawFileParser: Modular, Scalable, and Cross-Platform RAW File Conversion,” Journal of proteome research, vol. 19, iss. 1, p. 537–542, 2019.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{hulstaert2019thermorawfileparser,
title={ThermoRawFileParser: Modular, Scalable, and Cross-Platform RAW File Conversion},
author={Hulstaert, Niels and Shofstahl, Jim and Sachsenberg, Timo and Walzer, Mathias and Barsnes, Harald and Martens, Lennart and Perez-Riverol, Yasset},
journal={Journal of proteome research},
volume={19},
number={1},
pages={537--542},
year={2019},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>L. Bichmann, A. Nelde, M. Ghosh, L. Heumos, C. Mohr, A. Peltzer, L. Kuchenbecker, T. Sachsenberg, J. S. Walz, S. Stevanović, and others, “MHCquant: Automated and reproducible data analysis for immunopeptidomics,” Journal of proteome research, vol. 18, iss. 11, p. 3876–3884, 2019.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bichmann2019mhcquant,
title={MHCquant: Automated and reproducible data analysis for immunopeptidomics},
author={Bichmann, Leon and Nelde, Annika and Ghosh, Michael and Heumos, Lukas and Mohr, Christopher and Peltzer, Alexander and Kuchenbecker, Leon and Sachsenberg, Timo and Walz, Juliane S and Stevanović, Stefan and others},
journal={Journal of proteome research},
volume={18},
number={11},
pages={3876--3884},
year={2019},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>M. W. Löffler, C. Mohr, L. Bichmann, L. K. Freudenmann, M. Walzer, C. M. Schroeder, N. Trautwein, F. J. Hilke, R. S. Zinser, L. Mühlenbruch, and others, “Multi-omics discovery of exome-derived neoantigens in hepatocellular carcinoma,” Genome medicine, vol. 11, iss. 1, p. 28, 2019.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{loffler2019multi,
title={Multi-omics discovery of exome-derived neoantigens in hepatocellular carcinoma},
author={L{\"o}ffler, Markus W and Mohr, Christopher and Bichmann, Leon and Freudenmann, Lena Katharina and Walzer, Mathias and Schroeder, Christopher M and Trautwein, Nico and Hilke, Franz J and Zinser, Raphael S and M{\"u}hlenbruch, Lena and others},
journal={Genome medicine},
volume={11},
number={1},
pages={28},
year={2019},
publisher={BioMed Central}
}
</code></pre></details><hr><ul><li>A. Marcu, L. Bichmann, L. Kuchenbecker, L. Backert, D. J. Kowalewski, L. K. Freudenmann, M. W. Löffler, M. Lübke, J. S. Walz, J. Velz, and others, “The HLA Ligand Atlas. A resource of natural HLA ligands presented on benign tissues,” BioRxiv, p. 778944, 2019.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{marcu2019hla,
title={The HLA Ligand Atlas. A resource of natural HLA ligands presented on benign tissues},
author={Marcu, Ana and Bichmann, Leon and Kuchenbecker, Leon and Backert, Linus and Kowalewski, Daniel J and Freudenmann, Lena Katharina and L{\"o}ffler, Markus W and L{\"u}bke, Maren and Walz, Juliane S and Velz, Julia and others},
journal={BioRxiv},
pages={778944},
year={2019},
publisher={Cold Spring Harbor Laboratory}
}
</code></pre></details><hr><h3 id=2018>2018</h3><ul><li>B. Gruening, O. Sallou, P. Moreno, F. da Veiga Leprevost, H. Ménager, D. S{o}ndergaard, H. Röst, T. Sachsenberg, B. O’Connor, F. Madeira, and others, “Recommendations for the packaging and containerizing of bioinformatics software,” F1000Research, vol. 7, 2018.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{gruening2018recommendations,
title={Recommendations for the packaging and containerizing of bioinformatics software},
author={Gruening, Bjorn and Sallou, Olivier and Moreno, Pablo and da Veiga Leprevost, Felipe and M{\'e}nager, Herv{\'e} and S{\o}ndergaard, Dan and R{\"o}st, Hannes and Sachsenberg, Timo and O'Connor, Brian and Madeira, F{\'a}bio and others},
journal={F1000Research},
volume={7},
year={2018}
}
</code></pre></details><hr><ul><li>A. Kahles, K. Lehmann, N. C. Toussaint, M. Hüser, S. G. Stark, T. Sachsenberg, O. Stegle, O. Kohlbacher, C. Sander, S. J. Caesar-Johnson, and others, “Comprehensive analysis of alternative splicing across tumors from 8,705 patients,” Cancer cell, vol. 34, iss. 2, p. 211–224, 2018.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{kahles2018comprehensive,
title={Comprehensive analysis of alternative splicing across tumors from 8,705 patients},
author={Kahles, Andr{\'e} and Lehmann, Kjong-Van and Toussaint, Nora C and H{\"u}ser, Matthias and Stark, Stefan G and Sachsenberg, Timo and Stegle, Oliver and Kohlbacher, Oliver and Sander, Chris and Caesar-Johnson, Samantha J and others},
journal={Cancer cell},
volume={34},
number={2},
pages={211--224},
year={2018},
publisher={Elsevier}
}
</code></pre></details><hr><h3 id=2017>2017</h3><ul><li>J. A. Vizcaíno, G. Mayer, S. R. Perkins, H. Barsnes, M. Vaudel, Y. Perez-Riverol, T. Ternent, J. Uszkoreit, M. Eisenacher, L. Fischer, and others, “The mzIdentML data standard version 1.2, supporting advances in proteome informatics,” Molecular & Cellular Proteomics, p. mcp–M117, 2017.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{vizcaino2017mzidentml,
title={The mzIdentML data standard version 1.2, supporting advances in proteome informatics},
author={Vizca{\'i}no, Juan Antonio and Mayer, Gerhard and Perkins, Simon R and Barsnes, Harald and Vaudel, Marc and Perez-Riverol, Yasset and Ternent, Tobias and Uszkoreit, Julian and Eisenacher, Martin and Fischer, Lutz and others},
journal={Molecular \& Cellular Proteomics},
pages={mcp--M117},
year={2017},
publisher={ASBMB}
}
</code></pre></details><hr><ul><li>E. Audain, J. Uszkoreit, T. Sachsenberg, J. Pfeuffer, X. Liang, H. Hermjakob, A. Sanchez, M. Eisenacher, K. Reinert, D. L. Tabb, and others, “In-depth analysis of protein inference algorithms using multiple search engines and well-defined metrics,” Journal of proteomics, vol. 150, p. 170–182, 2017.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{audain2017depth,
title={In-depth analysis of protein inference algorithms using multiple search engines and well-defined metrics},
author={Audain, Enrique and Uszkoreit, Julian and Sachsenberg, Timo and Pfeuffer, Julianus and Liang, Xiao and Hermjakob, Henning and Sanchez, Aniel and Eisenacher, Martin and Reinert, Knut and Tabb, David L and others},
journal={Journal of proteomics},
volume={150},
pages={170--182},
year={2017},
publisher={Elsevier}
}
</code></pre></details><hr><ul><li>H. L. Röst, R. Aebersold, and O. T. Schubert, “Automated SWATH data analysis using targeted extraction of ion chromatograms,” Proteomics: Methods and Protocols, p. 289–307, 2017.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{rost2017automated,
title={Automated SWATH data analysis using targeted extraction of ion chromatograms},
author={R{\"o}st, Hannes L and Aebersold, Ruedi and Schubert, Olga T},
journal={Proteomics: Methods and Protocols},
pages={289--307},
year={2017},
publisher={Springer New York}
}
</code></pre></details><hr><ul><li>F. da Veiga Leprevost, B. A. Grüning, S. Alves Aflitos, H. L. Röst, J. Uszkoreit, H. Barsnes, M. Vaudel, P. Moreno, L. Gatto, J. Weber, and others, “BioContainers: an open-source and community-driven framework for software standardization,” Bioinformatics, p. btx192, 2017.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{da2017biocontainers,
title={BioContainers: an open-source and community-driven framework for software standardization},
author={da Veiga Leprevost, Felipe and Gr{\"u}ning, Bj{\"o}rn A and Alves Aflitos, Saulo and R{\"o}st, Hannes L and Uszkoreit, Julian and Barsnes, Harald and Vaudel, Marc and Moreno, Pablo and Gatto, Laurent and Weber, Jonas and others},
journal={Bioinformatics},
pages={btx192},
year={2017},
publisher={Oxford University Press}
}
</code></pre></details><hr><ul><li>H. Weisser and J. S. Choudhary, “Targeted feature detection for data-dependent shotgun proteomics,” Journal of Proteome Research, 2017.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{weisser2017targeted,
title={Targeted feature detection for data-dependent shotgun proteomics},
author={Weisser, Hendrik and Choudhary, Jyoti S},
journal={Journal of Proteome Research},
year={2017},
publisher={ACS Publications},
doi={10.1021/acs.jproteome.7b00248}
}
</code></pre></details><hr><ul><li>J. Pfeuffer, T. Sachsenberg, O. Alka, M. Walzer, A. Fillbrunn, L. Nilse, O. Schilling, K. Reinert, and O. Kohlbacher, “OpenMS – A platform for reproducible analysis of mass spectrometry data,” Journal of Biotechnology, 2017.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{pfeuffer2017denbi,
title = "OpenMS – A platform for reproducible analysis of mass spectrometry data",
journal = "Journal of Biotechnology",
volume = "",
number = "",
pages = "",
year = "2017",
note = "",
issn = "0168-1656",
doi = "10.1016/j.jbiotec.2017.05.016",
url = "http://www.sciencedirect.com/science/article/pii/S0168165617302511",
author = "Julianus Pfeuffer and Timo Sachsenberg and Oliver Alka and Mathias Walzer and Alexander Fillbrunn and Lars Nilse and Oliver Schilling and Knut Reinert and Oliver Kohlbacher"
}
</code></pre></details><hr><h3 id=2016>2016</h3><ul><li>H. L. Rost, T. Sachsenberg, S. Aiche, C. Bielow, H. Weisser, F. Aicheler, S. Andreotti, H. Ehrlich, P. Gutenbrunner, E. Kenar, X. Liang, S. Nahnsen, L. Nilse, J. Pfeuffer, G. Rosenberger, M. Rurik, U. Schmitt, J. Veit, M. Walzer, D. Wojnar, W. E. Wolski, O. Schilling, J. S. Choudhary, L. Malmstrom, R. Aebersold, K. Reinert, and O. Kohlbacher, “OpenMS: a flexible open-source software platform for mass spectrometry data analysis,” Nat Meth, vol. 13, iss. 9, pp. 741-748, 2016.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{Rost2016,
author = {Rost, Hannes L and Sachsenberg, Timo and Aiche, Stephan and Bielow, Chris and Weisser, Hendrik and Aicheler, Fabian and Andreotti, Sandro and Ehrlich, Hans-Christian and Gutenbrunner, Petra and Kenar, Erhan and Liang, Xiao and Nahnsen, Sven and Nilse, Lars and Pfeuffer, Julianus and Rosenberger, George and Rurik, Marc and Schmitt, Uwe and Veit, Johannes and Walzer, Mathias and Wojnar, David and Wolski, Witold E and Schilling, Oliver and Choudhary, Jyoti S and Malmstrom, Lars and Aebersold, Ruedi and Reinert, Knut and Kohlbacher, Oliver},
title = {OpenMS: a flexible open-source software platform for mass spectrometry data analysis},
year = {2016},
URL = {http://dx.doi.org/10.1038/nmeth.3959},
abstract = {High-resolution mass spectrometry (MS) has become an important tool in the life sciences, contributing to the diagnosis and understanding of human diseases, elucidating biomolecular structural information and characterizing cellular signaling networks. However, the rapid growth in the volume and complexity of MS data makes transparent, accurate and reproducible analysis difficult. We present OpenMS 2.0 (http://www.openms.de), a robust, open-source, cross-platform software specifically designed for the flexible and reproducible analysis of high-throughput MS data. The extensible OpenMS software implements common mass spectrometric data processing tasks through a well-defined application programming interface in C++ and Python and through standardized open data formats. OpenMS additionally provides a set of 185 tools and ready-made workflows for common mass spectrometric data processing tasks, which enable users to perform complex quantitative mass spectrometric analyses with ease.},
journal = {Nat Meth},
volume = {13},
number = {9},
pages = {741-748},
note = {Perspective}
}
</code></pre></details><hr><ul><li>J. Veit, T. Sachsenberg, A. Chernev, F. Aicheler, H. Urlaub, and O. Kohlbacher, “LFQProfiler and RNPxl: Open-Source Tools for Label-Free Quantification and Protein–RNA Cross-Linking Integrated into Proteome Discoverer,” Journal of Proteome Research, vol. 15, iss. 9, p. 3441–3448, 2016.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{veit2016lfqprofiler,
title={LFQProfiler and RNPxl: Open-Source Tools for Label-Free Quantification and Protein--RNA Cross-Linking Integrated into Proteome Discoverer},
author={Veit, Johannes and Sachsenberg, Timo and Chernev, Aleksandar and Aicheler, Fabian and Urlaub, Henning and Kohlbacher, Oliver},
journal={Journal of Proteome Research},
volume={15},
number={9},
pages={3441--3448},
year={2016},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>P. Navarro, J. Kuharev, L. C. Gillet, O. M. Bernhardt, B. MacLean, H. L. Röst, S. A. Tate, C. Tsou, L. Reiter, U. Distler, and others, “A multi-center study benchmarks software tools for label-free proteome quantification,” Nature biotechnology, vol. 34, iss. 11, p. 1130, 2016.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{navarro2016multi,
title={A multi-center study benchmarks software tools for label-free proteome quantification},
author={Navarro, Pedro and Kuharev, J{\"o}rg and Gillet, Ludovic C and Bernhardt, Oliver M and MacLean, Brendan and R{\"o}st, Hannes L and Tate, Stephen A and Tsou, Chih-Chiang and Reiter, Lukas and Distler, Ute and others},
journal={Nature biotechnology},
volume={34},
number={11},
pages={1130},
year={2016},
publisher={Europe PMC Funders}
}
</code></pre></details><hr><ul><li>H. Weisser, J. C. Wright, J. M. Mudge, P. Gutenbrunner, and J. S. Choudhary, “Flexible Data Analysis Pipeline for High-Confidence Proteogenomics,” Journal of Proteome Research, vol. 15, iss. 12, p. 4686–4695, 2016.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{weisser2016flexible,
title={Flexible Data Analysis Pipeline for High-Confidence Proteogenomics},
author={Weisser, Hendrik and Wright, James C and Mudge, Jonathan M and Gutenbrunner, Petra and Choudhary, Jyoti S},
journal={Journal of Proteome Research},
volume={15},
number={12},
pages={4686--4695},
year={2016},
publisher={ACS Publications},
doi={10.1021/acs.jproteome.6b00765}
}
</code></pre></details><hr><h3 id=2015>2015</h3><ul><li>S. Aiche, T. Sachsenberg, E. Kenar, M. Walzer, B. Wiswedel, T. Kristl, M. Boyles, A. Duschl, C. G. Huber, M. R. Berthold, and others, “Workflows for automated downstream data analysis and visualization in large-scale computational mass spectrometry,” Proteomics, vol. 15, iss. 8, p. 1443–1447, 2015.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{aiche2015workflows,
title={Workflows for automated downstream data analysis and visualization in large-scale computational mass spectrometry},
author={Aiche, Stephan and Sachsenberg, Timo and Kenar, Erhan and Walzer, Mathias and Wiswedel, Bernd and Kristl, Theresa and Boyles, Matthew and Duschl, Albert and Huber, Christian G and Berthold, Michael R and others},
journal={Proteomics},
volume={15},
number={8},
pages={1443--1447},
year={2015}
}
</code></pre></details><hr><ul><li>F. Aicheler, J. Li, M. Hoene, R. Lehmann, G. Xu, and O. Kohlbacher, “Retention Time Prediction Improves Identification in Nontargeted Lipidomics Approaches,” Analytical chemistry, vol. 87, iss. 15, p. 7698–7704, 2015.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{aicheler2015retention,
title={Retention Time Prediction Improves Identification in Nontargeted Lipidomics Approaches},
author={Aicheler, Fabian and Li, Jia and Hoene, Miriam and Lehmann, Rainer and Xu, Guowang and Kohlbacher, Oliver},
journal={Analytical chemistry},
volume={87},
number={15},
pages={7698--7704},
year={2015},
publisher={American Chemical Society}
}
</code></pre></details><hr><ul><li>C. Ranninger, M. Rurik, A. Limonciel, S. Ruzek, R. Reischl, A. Wilmes, P. Jennings, P. Hewitt, W. Dekant, O. Kohlbacher, and others, “Nephron Toxicity Profiling via Untargeted Metabolome Analysis Employing a High Performance Liquid Chromatography-Mass Spectrometry-based Experimental and Computational Pipeline,” Journal of Biological Chemistry, vol. 290, iss. 31, p. 19121–19132, 2015.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{ranninger2015nephron,
title={Nephron Toxicity Profiling via Untargeted Metabolome Analysis Employing a High Performance Liquid Chromatography-Mass Spectrometry-based Experimental and Computational Pipeline},
author={Ranninger, Christina and Rurik, Marc and Limonciel, Alice and Ruzek, Silke and Reischl, Roland and Wilmes, Anja and Jennings, Paul and Hewitt, Philip and Dekant, Wolfgang and Kohlbacher, Oliver and others},
journal={Journal of Biological Chemistry},
volume={290},
number={31},
pages={19121--19132},
year={2015},
publisher={ASBMB}
}
</code></pre></details><hr><ul><li>H. L. Röst, U. Schmitt, R. Aebersold, and L. Malmström, “Fast and efficient XML data access for next-generation mass spectrometry,” PloS one, vol. 10, iss. 4, p. e0125108, 2015.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{rost2015fast,
title={Fast and efficient XML data access for next-generation mass spectrometry},
author={R{\"o}st, Hannes L and Schmitt, Uwe and Aebersold, Ruedi and Malmstr{\"o}m, Lars},
journal={PloS one},
volume={10},
number={4},
pages={e0125108},
year={2015},
publisher={Public Library of Science}
}
</code></pre></details><hr><ul><li>H. L. Röst, G. Rosenberger, R. Aebersold, and L. Malmström, “Efficient visualization of high-throughput targeted proteomics experiments: TAPIR,” Bioinformatics, p. btv152, 2015.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{rost2015efficient,
title={Efficient visualization of high-throughput targeted proteomics experiments: TAPIR},
author={R{\"o}st, Hannes L and Rosenberger, George and Aebersold, Ruedi and Malmstr{\"o}m, Lars},
journal={Bioinformatics},
pages={btv152},
year={2015},
publisher={Oxford Univ Press}
}
</code></pre></details><hr><ul><li>L. Nilse, F. C. Sigloch, M. L. Biniossek, and O. Schilling, “Toward improved peptide feature detection in quantitative proteomics using stable isotope labeling,” PROTEOMICS-Clinical Applications, vol. 9, iss. 7-8, p. 706–714, 2015.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{nilse2015toward,
title={Toward improved peptide feature detection in quantitative proteomics using stable isotope labeling},
author={Nilse, Lars and Sigloch, Florian Christoph and Biniossek, Martin L and Schilling, Oliver},
journal={PROTEOMICS-Clinical Applications},
volume={9},
number={7-8},
pages={706--714},
year={2015}
}
</code></pre></details><hr><h3 id=2014>2014</h3><ul><li>H. L. Röst, G. Rosenberger, P. Navarro, L. Gillet, S. M. Miladinović, O. T. Schubert, W. Wolski, B. C. Collins, J. Malmström, L. Malmström, and others, “OpenSWATH enables automated, targeted analysis of data-independent acquisition MS data,” Nature biotechnology, vol. 32, iss. 3, p. 219–223, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{rost2014openswath,
title={OpenSWATH enables automated, targeted analysis of data-independent acquisition MS data},
author={R{\"o}st, Hannes L and Rosenberger, George and Navarro, Pedro and Gillet, Ludovic and Miladinovi{\'c}, Sa{\v{s}}a M and Schubert, Olga T and Wolski, Witold and Collins, Ben C and Malmstr{\"o}m, Johan and Malmstr{\"o}m, Lars and others},
journal={Nature biotechnology},
volume={32},
number={3},
pages={219--223},
year={2014},
publisher={Nature Publishing Group}
}
</code></pre></details><hr><ul><li>H. L. Röst, U. Schmitt, R. Aebersold, and L. Malmström, “pyOpenMS: A Python-based interface to the OpenMS mass-spectrometry algorithm library,” Proteomics, vol. 14, iss. 1, p. 74–77, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{rost2014pyopenms,
title={pyOpenMS: A Python-based interface to the OpenMS mass-spectrometry algorithm library},
author={R{\"o}st, Hannes L and Schmitt, Uwe and Aebersold, Ruedi and Malmstr{\"o}m, Lars},
journal={Proteomics},
volume={14},
number={1},
pages={74--77},
year={2014},
publisher={Wiley Online Library}
}
</code></pre></details><hr><ul><li>E. Kenar, H. Franken, S. Forcisi, K. Wörmann, H. Häring, R. Lehmann, P. Schmitt-Kopplin, A. Zell, and O. Kohlbacher, “Automated Label-free Quantification of Metabolites from Liquid Chromatography–Mass Spectrometry Data,” Molecular & Cellular Proteomics, vol. 13, iss. 1, p. 348–359, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{kenar2014automated,
title={Automated Label-free Quantification of Metabolites from Liquid Chromatography--Mass Spectrometry Data},
author={Kenar, Erhan and Franken, Holger and Forcisi, Sara and W{\"o}rmann, Kilian and H{\"a}ring, Hans-Ulrich and Lehmann, Rainer and Schmitt-Kopplin, Philippe and Zell, Andreas and Kohlbacher, Oliver},
journal={Molecular \& Cellular Proteomics},
volume={13},
number={1},
pages={348--359},
year={2014},
publisher={ASBMB}
}
</code></pre></details><hr><ul><li>T. Sachsenberg, F. Herbst, M. Taubert, R. Kermer, N. Jehmlich, M. von Bergen, J. Seifert, and O. Kohlbacher, “MetaProSIP: automated inference of stable isotope incorporation rates in proteins for functional metaproteomics,” Journal of proteome research, vol. 14, iss. 2, p. 619–627, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{sachsenberg2014metaprosip,
title={MetaProSIP: automated inference of stable isotope incorporation rates in proteins for functional metaproteomics},
author={Sachsenberg, Timo and Herbst, Florian-Alexander and Taubert, Martin and Kermer, Ren{\'e} and Jehmlich, Nico and von Bergen, Martin and Seifert, Jana and Kohlbacher, Oliver},
journal={Journal of proteome research},
volume={14},
number={2},
pages={619--627},
year={2014},
publisher={American Chemical Society}
}
</code></pre></details><hr><ul><li>K. Kramer, T. Sachsenberg, B. M. Beckmann, S. Qamar, K. Boon, M. W. Hentze, O. Kohlbacher, and H. Urlaub, “Photo-cross-linking and high-resolution mass spectrometry for assignment of RNA-binding sites in RNA-binding proteins,” Nature methods, vol. 11, iss. 10, p. 1064–1070, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{kramer2014photo,
title={Photo-cross-linking and high-resolution mass spectrometry for assignment of RNA-binding sites in RNA-binding proteins},
author={Kramer, Katharina and Sachsenberg, Timo and Beckmann, Benedikt M and Qamar, Saadia and Boon, Kum-Loong and Hentze, Matthias W and Kohlbacher, Oliver and Urlaub, Henning},
journal={Nature methods},
volume={11},
number={10},
pages={1064--1070},
year={2014},
publisher={Nature Publishing Group}
}
</code></pre></details><hr><ul><li>J. Griss, A. R. Jones, T. Sachsenberg, M. Walzer, L. Gatto, J. Hartler, G. G. Thallinger, R. M. Salek, C. Steinbeck, N. Neuhauser, and others, “The mzTab data exchange format: communicating mass-spectrometry-based proteomics and metabolomics experimental results to a wider audience,” Molecular & Cellular Proteomics, vol. 13, iss. 10, p. 2765–2775, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{griss2014mztab,
title={The mzTab data exchange format: communicating mass-spectrometry-based proteomics and metabolomics experimental results to a wider audience},
author={Griss, Johannes and Jones, Andrew R and Sachsenberg, Timo and Walzer, Mathias and Gatto, Laurent and Hartler, J{\"u}rgen and Thallinger, Gerhard G and Salek, Reza M and Steinbeck, Christoph and Neuhauser, Nadin and others},
journal={Molecular \& Cellular Proteomics},
volume={13},
number={10},
pages={2765--2775},
year={2014},
publisher={ASBMB}
}
</code></pre></details><hr><ul><li>M. Walzer, L. E. Pernas, S. Nasso, W. Bittremieux, S. Nahnsen, P. Kelchtermans, P. Pichler, H. W. van den Toorn, A. Staes, J. Vandenbussche, and others, “qcML: an exchange format for quality control metrics from mass spectrometry experiments,” Molecular & Cellular Proteomics, vol. 13, iss. 8, p. 1905–1913, 2014.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{walzer2014qcml,
title={qcML: an exchange format for quality control metrics from mass spectrometry experiments},
author={Walzer, Mathias and Pernas, Lucia Espona and Nasso, Sara and Bittremieux, Wout and Nahnsen, Sven and Kelchtermans, Pieter and Pichler, Peter and van den Toorn, Henk WP and Staes, An and Vandenbussche, Jonathan and others},
journal={Molecular \& Cellular Proteomics},
volume={13},
number={8},
pages={1905--1913},
year={2014},
publisher={American Society for Biochemistry and Molecular Biology}
}
</code></pre></details><hr><h3 id=2013>2013</h3><ul><li>H. Weisser, S. Nahnsen, J. Grossmann, L. Nilse, A. Quandt, H. Brauer, M. Sturm, E. Kenar, O. Kohlbacher, R. Aebersold, and others, “An automated pipeline for high-throughput label-free quantitative proteomics,” Journal of proteome research, vol. 12, iss. 4, p. 1628–1644, 2013.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{weisser2013automated,
title={An automated pipeline for high-throughput label-free quantitative proteomics},
author={Weisser, Hendrik and Nahnsen, Sven and Grossmann, Jonas and Nilse, Lars and Quandt, Andreas and Brauer, Hendrik and Sturm, Marc and Kenar, Erhan and Kohlbacher, Oliver and Aebersold, Ruedi and others},
journal={Journal of proteome research},
volume={12},
number={4},
pages={1628--1644},
year={2013},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>M. Walzer, D. Qi, G. Mayer, J. Uszkoreit, M. Eisenacher, T. Sachsenberg, F. F. Gonzalez-Galarza, J. Fan, C. Bessant, E. W. Deutsch, and others, “The mzQuantML Data Standard for Mass Spectrometry–based Quantitative Studies in Proteomics,” Molecular & Cellular Proteomics, vol. 12, iss. 8, p. 2332–2340, 2013.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{walzer2013mzquantml,
title={The mzQuantML Data Standard for Mass Spectrometry--based Quantitative Studies in Proteomics},
author={Walzer, Mathias and Qi, Da and Mayer, Gerhard and Uszkoreit, Julian and Eisenacher, Martin and Sachsenberg, Timo and Gonzalez-Galarza, Faviel F and Fan, Jun and Bessant, Conrad and Deutsch, Eric W and others},
journal={Molecular \& Cellular Proteomics},
volume={12},
number={8},
pages={2332--2340},
year={2013},
publisher={ASBMB}
}
</code></pre></details><hr><h3 id=2012>2012</h3><ul><li>J. Junker, C. Bielow, A. Bertsch, M. Sturm, K. Reinert, and O. Kohlbacher, “TOPPAS: a graphical workflow editor for the analysis of high-throughput proteomics data,” Journal of proteome research, vol. 11, iss. 7, p. 3914–3920, 2012.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{junker2012toppas,
title={TOPPAS: a graphical workflow editor for the analysis of high-throughput proteomics data},
author={Junker, Johannes and Bielow, Chris and Bertsch, Andreas and Sturm, Marc and Reinert, Knut and Kohlbacher, Oliver},
journal={Journal of proteome research},
volume={11},
number={7},
pages={3914--3920},
year={2012},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>S. Nahnsen and O. Kohlbacher, “In silico design of targeted SRM-based experiments,” BMC bioinformatics, vol. 13, iss. Suppl 16, p. S8, 2012.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{nahnsen2012silico,
title={In silico design of targeted SRM-based experiments},
author={Nahnsen, Sven and Kohlbacher, Oliver},
journal={BMC bioinformatics},
volume={13},
number={Suppl 16},
pages={S8},
year={2012},
publisher={BioMed Central Ltd}
}
</code></pre></details><hr><h3 id=2011>2011</h3><ul><li>A. Bertsch, C. Gröpl, K. Reinert, and O. Kohlbacher, “OpenMS and TOPP: open source software for LC-MS data analysis,” Data Mining in Proteomics: From Standards to Applications, p. 353–367, 2011.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bertsch2011openms,
title={OpenMS and TOPP: open source software for LC-MS data analysis},
author={Bertsch, Andreas and Gr{\"o}pl, Clemens and Reinert, Knut and Kohlbacher, Oliver},
journal={Data Mining in Proteomics: From Standards to Applications},
pages={353--367},
year={2011},
publisher={Humana Press}
}
</code></pre></details><hr><ul><li>S. Nahnsen, A. Bertsch, J. Rahnenführer, A. Nordheim, and O. Kohlbacher, “Probabilistic consensus scoring improves tandem mass spectrometry peptide identification,” Journal of proteome research, vol. 10, iss. 8, p. 3332–3343, 2011.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{nahnsen2011probabilistic,
title={Probabilistic consensus scoring improves tandem mass spectrometry peptide identification},
author={Nahnsen, Sven and Bertsch, Andreas and Rahnenführer, Jörg and Nordheim, Alfred and Kohlbacher, Oliver},
journal={Journal of proteome research},
volume={10},
number={8},
pages={3332--3343},
year={2011},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>C. Bielow, S. Aiche, S. Andreotti, and K. Reinert, “MSSimulator: Simulation of mass spectrometry data,” Journal of proteome research, vol. 10, iss. 7, p. 2922–2929, 2011.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bielow2011mssimulator,
title={MSSimulator: Simulation of mass spectrometry data},
author={Bielow, Chris and Aiche, Stephan and Andreotti, Sandro and Reinert, Knut},
journal={Journal of proteome research},
volume={10},
number={7},
pages={2922--2929},
year={2011},
publisher={ACS Publications}
}
</code></pre></details><hr><h3 id=2010>2010</h3><ul><li>K. Reinert and O. Kohlbacher, “OpenMS and TOPP: open source software for LC-MS data analysis,” in Proteome Bioinformatics, Springer, 2010, p. 201–211.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@incollection{reinert2010openms,
title={OpenMS and TOPP: open source software for LC-MS data analysis},
author={Reinert, Knut and Kohlbacher, Oliver},
booktitle={Proteome Bioinformatics},
pages={201--211},
year={2010},
publisher={Springer}
}
</code></pre></details><hr><ul><li>M. Sturm and others, “OpenMS-A framework for computational mass spectrometry,” PhD Thesis, 2010.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@phdthesis{sturm2010openms,
title={OpenMS-A framework for computational mass spectrometry},
author={Sturm, Marc and others},
year={2010},
school={Universit{\"a}t T{\"u}bingen}
}
</code></pre></details><hr><ul><li>A. Bertsch, S. Jung, A. Zerck, N. Pfeifer, S. Nahnsen, C. Henneges, A. Nordheim, and O. Kohlbacher, “Optimal de novo design of MRM experiments for rapid assay development in targeted proteomics,” Journal of proteome research, vol. 9, iss. 5, p. 2696–2704, 2010.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bertsch2010optimal,
title={Optimal de novo design of MRM experiments for rapid assay development in targeted proteomics},
author={Bertsch, Andreas and Jung, Stephan and Zerck, Alexandra and Pfeifer, Nico and Nahnsen, Sven and Henneges, Carsten and Nordheim, Alfred and Kohlbacher, Oliver},
journal={Journal of proteome research},
volume={9},
number={5},
pages={2696--2704},
year={2010},
publisher={ACS Publications}
}
</code></pre></details><hr><h3 id=2009>2009</h3><ul><li>M. Sturm and O. Kohlbacher, “TOPPView: an open-source viewer for mass spectrometry data,” Journal of proteome research, vol. 8, iss. 7, p. 3760–3763, 2009.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{sturm2009toppview,
title={TOPPView: an open-source viewer for mass spectrometry data},
author={Sturm, Marc and Kohlbacher, Oliver},
journal={Journal of proteome research},
volume={8},
number={7},
pages={3760--3763},
year={2009},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>C. Bielow, S. Aiche, S. Andreotti, A. Bertsch, C. Gröpl, R. Hussong, N. Pfeifer, M. Sturm, A. Zerck, A. Hildebrandt, and others, “OpenMS—An Open Source Framework for Mass Spec-trometry Data,” German Conference on Bioinformatics 2009, p. 154.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bielow2009openms,
title={OpenMS—An Open Source Framework for Mass Spec-trometry Data},
author={Bielow, Chris and Aiche, Stephan and Andreotti, Sandro and Bertsch, Andreas and Gr{\"o}pl, Clemens and Hussong, Rene and Pfeifer, Nico and Sturm, Marc and Zerck, Alexandra and Hildebrandt, Andreas and others},
journal={German Conference on Bioinformatics 2009},
pages={154}
}
</code></pre></details><hr><ul><li>A. Bertsch, A. Leinenbach, A. Pervukhin, M. Lubeck, R. Hartmer, C. Baessmann, Y. A. Elnakady, R. Müller, S. Böcker, C. G. Huber, and others, “De novo peptide sequencing by tandem MS using complementary CID and electron transfer dissociation,” Electrophoresis, vol. 30, iss. 21, p. 3736–3747, 2009.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{bertsch2009novo,
title={De novo peptide sequencing by tandem MS using complementary CID and electron transfer dissociation},
author={Bertsch, Andreas and Leinenbach, Andreas and Pervukhin, Anton and Lubeck, Markus and Hartmer, Ralf and Baessmann, Carsten and Elnakady, Yasser Abbas and M{\"u}ller, Rolf and B{\"o}cker, Sebastian and Huber, Christian G and others},
journal={Electrophoresis},
volume={30},
number={21},
pages={3736--3747},
year={2009},
publisher={Wiley Online Library}
}
</code></pre></details><hr><ul><li>N. Pfeifer, A. Leinenbach, C. G. Huber, and O. Kohlbacher, “Improving peptide identification in proteome analysis by a two-dimensional retention time filtering approach,” Journal of proteome research, vol. 8, iss. 8, p. 4109–4115, 2009.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{pfeifer2009improving,
title={Improving peptide identification in proteome analysis by a two-dimensional retention time filtering approach},
author={Pfeifer, Nico and Leinenbach, Andreas and Huber, Christian G and Kohlbacher, Oliver},
journal={Journal of proteome research},
volume={8},
number={8},
pages={4109--4115},
year={2009},
publisher={ACS Publications}
}
</code></pre></details><hr><ul><li>L. Nilse, M. Sturm, D. Trudgian, M. Salek, P. F. Sims, K. M. Carroll, and S. J. Hubbard, “SILACAnalyzer-a tool for differential quantitation of stable isotope derived data,” in Computational Intelligence Methods for Bioinformatics and Biostatistics, Springer Berlin Heidelberg, 2009, p. 45–55.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@incollection{nilse2009silacanalyzer,
title={SILACAnalyzer-a tool for differential quantitation of stable isotope derived data},
author={Nilse, Lars and Sturm, Marc and Trudgian, David and Salek, Mogjiborahman and Sims, Paul FG and Carroll, Kathleen M and Hubbard, Simon J},
booktitle={Computational Intelligence Methods for Bioinformatics and Biostatistics},
pages={45--55},
year={2009},
publisher={Springer Berlin Heidelberg}
}
</code></pre></details><hr><ul><li>O. Kohlbacher and K. Reinert, “OpenMS and TOPP: Open source software for lc-ms data analysis,” Proteome Bioinformatics, vol. 604, iss. 2, 2009.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{kohlbacher2009openms,
title={OpenMS and TOPP: Open source software for lc-ms data analysis},
author={Kohlbacher, O and Reinert, K},
journal={Proteome Bioinformatics},
volume={604},
number={2},
year={2009}
}
</code></pre></details><hr><h3 id=2008>2008</h3><ul><li>M. Sturm, A. Bertsch, C. Gröpl, A. Hildebrandt, R. Hussong, E. Lange, N. Pfeifer, O. Schulz-Trieglaff, A. Zerck, K. Reinert, and others, “OpenMS–an open-source software framework for mass spectrometry,” BMC bioinformatics, vol. 9, iss. 1, p. 163, 2008.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{sturm2008openms,
title={OpenMS--an open-source software framework for mass spectrometry},
author={Sturm, Marc and Bertsch, Andreas and Gr{\"o}pl, Clemens and Hildebrandt, Andreas and Hussong, Rene and Lange, Eva and Pfeifer, Nico and Schulz-Trieglaff, Ole and Zerck, Alexandra and Reinert, Knut and others},
journal={BMC bioinformatics},
volume={9},
number={1},
pages={163},
year={2008},
publisher={BioMed Central Ltd}
}
</code></pre></details><hr><ul><li>O. Schulz-Trieglaff, R. Hussong, C. Gröpl, A. Leinenbach, A. Hildebrandt, C. Huber, and K. Reinert, “Computational quantification of peptides from LC-MS data,” Journal of Computational Biology, vol. 15, iss. 7, p. 685–704, 2008.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{schulz2008computational,
title={Computational quantification of peptides from LC-MS data},
author={Schulz-Trieglaff, Ole and Hussong, Rene and Gr{\"o}pl, Clemens and Leinenbach, Andreas and Hildebrandt, Andreas and Huber, Christian and Reinert, Knut},
journal={Journal of Computational Biology},
volume={15},
number={7},
pages={685--704},
year={2008},
publisher={Mary Ann Liebert, Inc. 2 Madison Avenue Larchmont, NY 10538 USA}
}
</code></pre></details><hr><ul><li>E. Lange, “Analysis of mass spectrometric data: peak picking and map alignment,” Freie Universtaet Berlin, 2008.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{lange2008analysis,
title={Analysis of mass spectrometric data: peak picking and map alignment},
author={Lange, Eva},
journal={Freie Universtaet Berlin},
year={2008}
}
</code></pre></details><hr><ul><li>O. Schulz-Trieglaff, N. Pfeifer, C. Gröpl, O. Kohlbacher, and K. Reinert, “LC-MSsim–a simulation software for liquid chromatography mass spectrometry data,” BMC bioinformatics, vol. 9, iss. 1, p. 1, 2008.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{schulz2008lc,
title={LC-MSsim--a simulation software for liquid chromatography mass spectrometry data},
author={Schulz-Trieglaff, Ole and Pfeifer, Nico and Gr{\"o}pl, Clemens and Kohlbacher, Oliver and Reinert, Knut},
journal={BMC bioinformatics},
volume={9},
number={1},
pages={1},
year={2008},
publisher={BioMed Central}
}
</code></pre></details><hr><h3 id=2007>2007</h3><ul><li>O. Kohlbacher, K. Reinert, C. Gröpl, E. Lange, N. Pfeifer, O. Schulz-Trieglaff, and M. Sturm, “TOPP—the OpenMS proteomics pipeline,” Bioinformatics, vol. 23, iss. 2, p. e191–e197, 2007.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{kohlbacher2007topp,
title={TOPP—the OpenMS proteomics pipeline},
author={Kohlbacher, Oliver and Reinert, Knut and Gr{\"o}pl, Clemens and Lange, Eva and Pfeifer, Nico and Schulz-Trieglaff, Ole and Sturm, Marc},
journal={Bioinformatics},
volume={23},
number={2},
pages={e191--e197},
year={2007},
publisher={Oxford Univ Press}
}
</code></pre></details><hr><ul><li>E. Lange, C. Gröpl, O. Schulz-Trieglaff, A. Leinenbach, C. Huber, and K. Reinert, “A geometric approach for the alignment of liquid chromatography—mass spectrometry data,” Bioinformatics, vol. 23, iss. 13, p. i273–i281, 2007.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{lange2007geometric,
title={A geometric approach for the alignment of liquid chromatography—mass spectrometry data},
author={Lange, Eva and Gr{\"o}pl, Clemens and Schulz-Trieglaff, Ole and Leinenbach, Andreas and Huber, Christian and Reinert, Knut},
journal={Bioinformatics},
volume={23},
number={13},
pages={i273--i281},
year={2007},
publisher={Oxford Univ Press}
}
</code></pre></details><hr><h3 id=2006>2006</h3><ul><li>K. Reinert, O. Kohlbacher, E. Lange, O. Schulz-Trieglaff, M. Sturm, N. Pfeifer, and others, “OpenMS-A Framework for Quantitative HPLC/MS-Based Proteomics,” Dagstuhl Seminar Proceedings, 2006.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{reinert2006openms,
title={OpenMS-A Framework for Quantitative HPLC/MS-Based Proteomics},
author={Reinert, Knut and Kohlbacher, Oliver and Lange, Eva and Schulz-Trieglaff, Ole and Sturm, Marc and Pfeifer, Nico and others},
journal={Dagstuhl Seminar Proceedings},
year={2006},
publisher={Schloss Dagstuhl-Leibniz-Zentrum fr Informatik}
}
</code></pre></details><hr><ul><li>E. Lange, “High-Accuracy Peak Picking of Proteomics Data Using Wavelet Techniques Eva Lange, Clemens Gropl, Knut Reinert, Oliver Kohlbacher, and Andreas Hildebrandt Pacific Symposium on Biocomputing 11: 243-254 (2006),” Pacific Symposium on Biocomputing, vol. 11, p. 243–254, 2006.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{lange2006high,
title={High-Accuracy Peak Picking of Proteomics Data Using Wavelet Techniques Eva Lange, Clemens Gropl, Knut Reinert, Oliver Kohlbacher, and Andreas Hildebrandt Pacific Symposium on Biocomputing 11: 243-254 (2006)},
author={Lange, Eva},
journal={Pacific Symposium on Biocomputing},
volume={11},
pages={243--254},
year={2006}
}
</code></pre></details><hr><ul><li>E. Lange, C. Gröpl, O. Kohlbacher, and A. Hildebrandt, “High-accuracy peak picking of proteomics data,” Dagstuhl Seminar Proceedings, 2006.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{lange2006high,
title={High-accuracy peak picking of proteomics data},
author={Lange, Eva and Gr{\"o}pl, Clemens and Kohlbacher, Oliver and Hildebrandt, Andreas},
journal={Dagstuhl Seminar Proceedings},
year={2006},
publisher={Schloss Dagstuhl-Leibniz-Zentrum f{\"u}r Informatik}
}
</code></pre></details><hr><h3 id=2005>2005</h3><ul><li>E. Lange, K. Reinert, C. Groepl, O. Kohlbacher, M. Sturm, and A. Hildebrandt, “OPENMS; a generic open source framework for chromatography/MS-based proteomics,” in MOLECULAR & CELLULAR PROTEOMICS, 2005, p. S25–S25.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@inproceedings{lange2005openms,
title={OPENMS; a generic open source framework for chromatography/MS-based proteomics},
author={Lange, E and Reinert, K and Groepl, C and Kohlbacher, O and Sturm, M and Hildebrandt, A},
booktitle={MOLECULAR \& CELLULAR PROTEOMICS},
volume={4},
number={8},
pages={S25--S25},
year={2005},
organization={AMER SOC BIOCHEMISTRY MOLECULAR BIOLOGY INC 9650 ROCKVILLE PIKE, BETHESDA, MD 20814-3996 USA}
}
</code></pre></details><hr><ul><li>C. Gröpl, E. Lange, K. Reinert, O. Kohlbacher, M. Sturm, C. G. Huber, B. M. Mayr, and C. L. Klein, “Algorithms for the automated absolute quantification of diagnostic markers in complex proteomics samples,” in Computational Life Sciences, Springer, 2005, p. 151–162.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@incollection{gropl2005algorithms,
title={Algorithms for the automated absolute quantification of diagnostic markers in complex proteomics samples},
author={Gr{\"o}pl, Clemens and Lange, Eva and Reinert, Knut and Kohlbacher, Oliver and Sturm, Marc and Huber, Christian G and Mayr, Bettina M and Klein, Christoph L},
booktitle={Computational Life Sciences},
pages={151--162},
year={2005},
publisher={Springer}
}
</code></pre></details><hr><ul><li>C. Gröpl, A. Hildebrandt, O. Kohlbacher, E. Lange, S. Lövenich, and M. Sturm, “OpenMS-Software for Mass Spectrometry,” Poster presented at the MBI Workshop on Computational Proteomics and Mass Spectrometry, 2005.</li></ul><details><summary style="padding:0 0 0 25px">[Bibtex]</summary><pre><code>@article{gropl2005openms,
title={OpenMS-Software for Mass Spectrometry},
author={Gr{\"o}pl, C and Hildebrandt, A and Kohlbacher, O and Lange, E and L{\"o}venich, S and Sturm, M},
journal={Poster presented at the MBI Workshop on Computational Proteomics and Mass Spectrometry},
year={2005}
}
</code></pre></details><hr></div></div></div></section><div id=backtotop><a href=# id=backtotop-color></a></div><footer id=footer class=footer><div class="container is-max-widescreen"><div id=footer-columns class=columns><div class=footer-logo-column><p style=font-weight:bolder;font-size:xx-large;color:#f0f8ff>OpenMS</p></div><div class=link-column><div class=footer-column><div class=footer-header></div><ul class=link-list><li class=link-list><a class=footer-link href=/install>Install</a></li><li class=link-list><a class=footer-link href=https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/index.html>Documentation</a></li><li class=link-list><a class=footer-link href=/citing-openms>Citing OpenMS</a></li></ul></div></div><div class=link-column><div class=footer-column><div class=footer-header></div><ul class=link-list><li class=link-list><a class=footer-link href=/about>About us</a></li><li class=link-list><a class=footer-link href=/community>Community</a></li><li class=link-list><a class=footer-link href=/contribute>Contribute</a></li><li class=link-list><a class=footer-link href=/code-of-conduct>Code of conduct</a></li></ul></div></div><div class=link-column><div class=footer-column><div class=footer-header></div><ul class=link-list><li class=link-list><a class=footer-link href=/gethelp>Get help</a></li><li class=link-list><a class=footer-link href=/terms>Terms of use</a></li><li class=link-list><a class=footer-link href=/privacy>Privacy</a></li><li class=link-list><a class=footer-link href=/press-kit>Press kit</a></li><li class=link-list><a class=footer-link href=/jobs>Jobs</a></li></ul></div></div><div class=footer-actions><p>Sign up for the latest openms news, resources,<br>and more</p><form action="https://openms.us4.list-manage.com/subscribe/post?u=5ddd0d1d6e807900a8212481a&id=287fa4253c" method=post id=mc-embedded-subscribe-form name=mc-embedded-subscribe-form class="validate sign-up-container" target=_blank novalidate><div class=sign-up-image><svg class="icon mail-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M22 6c0-1.1-.9-2-2-2H4c-1.1.0-2 .9-2 2v12c0 1.1.9 2 2 2h16c1.1.0 2-.9 2-2V6zm-2 0-8 5-8-5h16zm0 12H4V8l8 5 8-5v10z"/></svg></div><input type=email name=EMAIL class="required email sign-up-input" id=mce-EMAIL aria-label="Input for email, press enter to submit" onkeypress='(event.which===13||event.keyCode===13||event.key==="Enter")&&sendThankYou()'><div class=submission-instructions>Press Enter</div><button class=signup-button onclick=sendThankYou() aria-label=Submit><svg class="icon sent-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M2.01 21 23 12 2.01 3 2 10l15 2-15 2z"/></svg></button><div id=mce-responses class=clear><div class=response id=mce-error-response style=display:none></div><div class=response id=mce-success-response style=display:none></div></div><div style=position:absolute;left:-5000px aria-hidden=true><input type=text name=b_5ddd0d1d6e807900a8212481a_287fa4253c tabindex=-1></div><div class=clear><input type=submit value=Subscribe name=subscribe id=mc-embedded-subscribe class=button style=display:none></div></form><div class=thank-you>Thank you! &#127881;</div><nav class="level is-mobile"><div class=social-media-icons><a class=level-item href=https://github.com/OpenMS/OpenMS aria-label=https://github.com/OpenMS/OpenMS><svg class="icon github-icon" viewBox="0 0 24 24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 .297c-6.63.0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577.0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93.0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176.0.0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22.0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22.0 1.606-.015 2.896-.015 3.286.0.315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/></svg></a><a class=level-item href=https://twitter.com/OpenMS_team aria-label=https://twitter.com/OpenMS_team><svg class="icon twitter-icon" viewBox="0 0 24 24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M23.953 4.57a10 10 0 01-2.825.775 4.958 4.958.0 002.163-2.723c-.951.555-2.005.959-3.127 1.184A4.92 4.92.0 0011.78 8.288C7.69 8.095 4.067 6.13 1.64 3.162A4.822 4.822.0 00.974 5.637c0 1.71.87 3.213 2.188 4.096A4.904 4.904.0 01.934 9.117v.06a4.923 4.923.0 003.946 4.827 4.996 4.996.0 01-2.212.085 4.936 4.936.0 004.604 3.417A9.867 9.867.0 011.17 19.611c-.39.0-.779-.023-1.17-.067a13.995 13.995.0 007.557 2.209c9.053.0 13.998-7.496 13.998-13.985.0-.21.0-.42-.015-.63A9.935 9.935.0 0024 4.59z"/></svg></a></div></nav><div class=copyright>&copy; 2022 OpenMS. All rights reserved.</div></div></div></div></footer><script src=https://code.jquery.com/jquery-3.6.0.min.js></script>
<script src=https://www.mattboldt.com/demos/typed-js/js/typed.custom.js></script>
<script type=text/javascript src=/js/bundle.min.js></script>
<script src=https://cdn.jsdelivr.net/npm/uikit@3.7.6/dist/js/uikit.min.js></script>
<link rel=stylesheet href=https://cdn.jsdelivr.net/npm/uikit@3.7.6/dist/css/uikit.min.css><script type=text/javascript>setupShortcuts(maxLevel=2)</script><script src=https://cdn.rawgit.com/tsucres/GithubManyfacedCards/0.3.0/dist/gmc.min.js></script>
<script type=application/javascript>var doNotTrack=!1;doNotTrack||(window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)},ga.l=+new Date,ga("create","UA-162151853-1","auto"),ga("send","pageview"))</script><script async src=https://www.google-analytics.com/analytics.js></script>
<script defer data-domain=openms.org src=https://views.scientific-python.org/js/script.js></script></body></html>